### PR TITLE
Wall bug and bullet bug

### PR DESCRIPTION
**Pull request by captaincrutches**

### CODE REVIEW

File: src/agents/Bullet.ts [hunk 1]
```diff
@@ -7,18 +7,17 @@
 
 namespace Game {
   export class Bullet extends PhysicsObject {
-    static DefaultVelocity: number = 100;
+    static DefaultVelocity: number = 1000;
```
Comment (BenTheElder) [inline src/agents/Bullet.ts]: how crazy is this?

**Comment by BenTheElder** [comment]
LGTM, though I'm curious about the order of magnitude change with bullet velocity.

File: src/agents/Bullet.ts (hunk 1 above)
Comment (captaincrutches) [inline src/agents/Bullet.ts]: It keeps the velocity the same, since we don't thrust anymore and now just use a constant velocity.  Because P2 physics.

Comment (BenTheElder) [inline src/agents/Bullet.ts]: Ok good to know, merging.

**Comment by BenTheElder** [comment]
Merged after discussion of P2 physics and velocity.

**Pull request merged**